### ISSUE TITLE
Critical failure for bracket in title

### ISSUE THREAD

**Issue by imolb** [fnep/mtv_dl#78]
Having a opening bracket in the title search results in critical error.

$mtv_dl list title="("

CRITICAL <class 'sqlite3.OperationalError'>: user-defined function raised exception                                      
                                File "/home/blomi/bin/mtv_dl", line 8, in <module>                                                            
                                  sys.exit(main())                                                                                            
                                           ^^^^^^      

(And thanks for this great tool!)

**Comment by fnep** [comment]
This is an interesting one. I will have a look next week. Sorry for the inconvenience.

**Comment by fnep** [comment]
The issue is that the value for "title" must be a regular expression, but this regular expression is invalid. I will implement a warning and switch to string match for this case:

Example:
```
2023-11-17T10:02:26  WARNING  Invalid regular expression '(' (using string match instead): missing ), unterminated subpattern at position 0                                                                                                                                                                        
```

**Comment by fnep** [comment]
Fixed in version 0.23.12
